flexget/ui/v2/__init__.py

```diff
@@ -14,10 +14,11 @@
 
 manager = None
 debug = False
-app_base = None
+exists = True
 
 ui_base = os.path.dirname(os.path.realpath(__file__))
```

Comment:
Can we make this configurable? If someone wants to run flexget from git, I don't think the solution should be to copy the ui folders. Maybe have some way to specify in the config where the dist folder is located?

Reply:
You can always setup a symlink. If you are really doing dev work on the WebUI you would run it separately anyway.